src/Zio.Tests/Zio.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;net8.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
This is because VS Code can't select other TFMs, so the first one is always picked. I wanted to debug on net8.0.